xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties

```diff
@@ -2768,11 +2768,11 @@ xe.panels.rights.users=Usuarios
 xe.panels.rights.usersexplanation=Este campo deber\u00EDa contener el identificador wiki de cada usuario al que desee aplicar los derechos. Por ejemplo, para usuarios no identificados deber\u00EDa usar XWiki.JohnDoe. XWiki.XWikiGuest
 xe.panels.rights.groups=Grupos
 xe.panels.rights.groupsexplanation=Este campo deber\u00EDa contener el nombre Wiki de cada grupo a los que desea que se le apliquen los derechos. XWiki.XWikiAllGroup representa el grupo de todos los usuarios con una cuenta en su Wiki que est\u00E1n autenticados.
-xe.panels.rights.groupsvirtualexplanation=xwiki:XWiki.XWikiAllGroup representa al grupo de todos los usuarios aut\u00E9nticados utilizando una cuenta global.
+xe.panels.rights.groupsvirtualexplanation=<em>xwiki:XWiki.XWikiAllGroup</em> representa el grupo de todos los usuarios autenticados que utilizan una cuenta global.
```

Comment:
Not sure it's good to put HTML in translations. I don't think that's correct.

Reply:
It's not great but that's how this key was designed according to the default version so not much to do with this translations.